run.py

```diff
@@ -444,10 +445,15 @@ def run(args):
     platform = args["--platform"]
     build = args.get("--build", None)
 
-    if build and build not in ["released"]:
-        base_url, docker_registry, docker_image, docker_tag = fetch_build_artifacts(
-            build, rhbuild, platform
-        )
+    if build:
+        if build not in ["released", "upstream"]:
+            base_url, docker_registry, docker_image, docker_tag = fetch_build_artifacts(
+                build, rhbuild, platform
+            )
+        elif build == "upstream":
+            base_url, docker_registry, docker_image, docker_tag = fetch_upstream_build(
+                build, rhbuild
```

Comment:
@anrao19 rhbuild [here](https://github.com/red-hat-storage/cephci/blob/fa710c14fb951f17e94f60bcc98e3eff1cb986a7/run.py#L112) mentioned it as ceph downstream version, Is it ok to use same vraible for upstream ceph version?